npm-shrinkwrap.json

```diff
@@ -2722,6 +2817,96 @@
           "from": "jed@0.5.4",
           "resolved": "https://registry.npmjs.org/jed/-/jed-0.5.4.tgz"
         },
+        "mozlog": {
+          "version": "2.0.3",
+          "from": "mozlog@2.0.3",
+          "resolved": "https://registry.npmjs.org/mozlog/-/mozlog-2.0.3.tgz",
+          "dependencies": {
+            "intel": {
+              "version": "1.1.0",
+              "from": "intel@>=1.0.0 <2.0.0",
```

Comment:
My previous comment should have been made somewhere around here to provide better context. (I put the other one at the end of the new mozlog block, which has zero context for my comment).